spec.bs

```diff
@@ -997,6 +1060,7 @@ Note: Previously, the `payment` extension allowed for the creation of
       USVString payeeOrigin;
       PaymentCurrencyAmount total;
       PaymentCredentialInstrument instrument;
+      PublicKeyCredentialParameters browserBoundPubKeyCredParams;
```

Comment:
Should this be a `sequence<PublicKeyCredentialParameters>` ? Right now its a single object.

Also, it shouldn't be under `// Only used for authentication.`; either add a new section with another comment, or (preferred, I think?) put it above between isPayment and rpId.